### DISCUSSION TITLE
Queries on releases made for out-of-support version on severe issues.

### DISCUSSION THREAD

**Discussion by cygoh** [vmware/clarity#6250] [General]
Hi Clarity Team,

We are currently using version 3.x of the framework. We understand the version is currently out-of-support and the recommendation would be to move to the latest version, if possible.

Unfortunately, we are unable to perform the upgrade due to the need in supporting the older browsers such as Internet Explorer, thus we need to remain on v3 until there is no longer a need to support the older browsers.

I understand there are plans to have releases made for EOS version in the event of severe issues (e.g. security) arising from what I have read under the section of support policies on the clarity site.

Could I just check under what circumstances would there actually be no support given for EOS version even if the issue is critical?

Thank you.
Jun Yang

**Comment by bbogdanov** [comment]
Hi @cygoh,

We ended the support for old browsers when we announced `v5`. From `v3` to `v5` on the Angular side there is only one component that was added and the rest are bug fixes and a11y improvements.

As team working on a `open-source` project we welcome and support people that would like to contribute to the project and that includes EOS version improvements and bug fixing. 

If there is a critical bug, let us know in our issue tracker and we would do our best to help you resolve it.
